test/clienthellotest.c

```diff
@@ -116,7 +116,13 @@ static int test_client_hello(int currtest)
                                     (unsigned char *)alpn_prots,
                                     sizeof(alpn_prots) - 1))))
             goto end;
-
+        /*
+         * Otherwise we need to make sure we have a small enough message to
+         * not need padding.
+         */
+        else if (!TEST_true(SSL_CTX_set_cipher_list(ctx,
```

Comment:
Does that else belong to just the "currtest == TEST_ADD_PADDING"?

Reply:
Yes, good catch. Fixed.